docs/userguide/main.md

```diff
@@ -311,6 +311,27 @@ Or custom ones, for example that Aiven plugin for prometheus-exporter:
     pluginsList: ["repository-s3","https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/1.3.0.0/prometheus-exporter-1.3.0.0.zip"]
 ```
 
+## Custom image repository
+A custom image repository can be configured as needed (for example an air gapped environment). Globally used image repository is set using the `DefaultRepo` option:
```

Comment:
NIT: please change the heading to Custom init helper and a line that says this image will be used during the cluster setup. :)

Reply:
I made some adjustments in it to make it more clear. Does it make sense to you?